include/nuttx/net/netconfig.h

```diff
@@ -456,11 +456,11 @@
  * connection requires approximately 30 bytes of memory.
  */
 
-#ifndef CONFIG_NET_TCP_CONNS
-#  ifdef CONFIG_NET_TCP
-#   define CONFIG_NET_TCP_CONNS 10
+#ifndef CONFIG_NET_TCP_PREALLOC_CONNS
```

Comment:
should we simply remove line 450-466?

Reply:
I think that this is temporarily needed, till `NET_ALLOC_CONNS` is removed.  
I kept this for this purpose, with the intention to remove this soon.

Reply:
If so, let's simplify to:
```
#ifndef CONFIG_NET_TCP_PREALLOC_CONNS
#  define CONFIG_NET_TCP_PREALLOC_CONNS 0
#endif
```